run-tests.sh

```diff
@@ -16,7 +16,7 @@
 
 set -e
 
-TESTS_EXTRAS_URL="https://git.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git/tests-extras"
```

Comment:
don't know if this is used, but the tool was moved to the new repo, the old one is deprecated